int32 id
string category
int32 age
